taxi_service/settings.py

```diff
@@ -56,7 +56,7 @@
 TEMPLATES = [
     {
         "BACKEND": "django.template.backends.django.DjangoTemplates",
-        "DIRS": [],
+        "DIRS": [BASE_DIR, "templates"],
```

Comment:
It's not recommended to add the BASE_DIR to the DIRS option of the TEMPLATES setting. This could potentially expose all your server files in your templates. It's better to specify the exact path to the templates directory, like `BASE_DIR / 'templates'`.